esmvaltool/cmorizers/data/formatters/datasets/cds_xch4.ncl

```diff
@@ -23,12 +23,12 @@
 ;
 ; ############################################################################
 loadscript(getenv("esmvaltool_root") + \
-           "/esmvaltool/cmorizers/obs/interface.ncl")
+           "/data/formatters/interface.ncl")
 
 begin
 
   ; Script name (for logger)
-  DIAG_SCRIPT = "cmorize_obs_cds_xch4.ncl"
+  DIAG_SCRIPT = "cds_xch4.ncl"
 
   ; Source name
   OBSNAME = "CDS-XCH4"
```

Comment:
There is an inconsistency between the name of the file downloaded and what is expected as input here.

Reply:
It should be fixed now (and there are two more years as a bonus)